cogni-libs/src/main/java/zone/cogni/libs/sparqlservice/impl/FusekiSparqlService.java

```diff
@@ -76,35 +107,55 @@ public void executeUpdateQuery(String updateQuery) {
 
   @Override
   public void upload(Model model, String graphUri) {
```

Comment:
is this upload replacing graph or updating with new values? one method we made in virtuoso became very handy with replace graph on demand parameter `public void upload(Model model, String graphUri, boolean replace)`